src/trio/_tools/gen_exports.py

```diff
@@ -184,8 +213,8 @@ def gen_public_wrappers_source(file: File) -> str:
 
     generated = ["".join(header)]
 
-    source = astor.code_to_ast.parse_file(file.path)
-    method_names = []
+    source = ast.parse(file.path.read_text())
```

Comment:
Is there a reason why we're not using `astor` anymore?

If it's because we're running this through an autoformatter which should do AST -> code, then maybe we could use `ast.unparse` instead of `astor.to_source` and drop `astor` entirely. Though I see that `ast.unparse` is 3.9+ (cause type hints...) so I guess maybe that's just a `# TODO: ...` comment for when we drop 3.8.

Reply:
> Is there a reason why we're not using astor anymore?

astor is untyped. This function is literally just a wrapper around `ast.parse` that opens the file. Errors in my editor annoy me.